zeebe/test-util/src/main/java/io/camunda/zeebe/test/util/junit/CachedTestResultsExtension.java

```diff
@@ -249,6 +255,7 @@ static InvocationCache recover(final Path path) {
             reader
                 .lines()
                 .map(CachedInvocation::fromLine)
+                .<CachedInvocation>mapMulti(Optional::ifPresent)
```

Comment:
ℹ️ Just learned about `mapMulti` today. Some comparison to `flatMap` can be found here: https://nipafx.dev/java-16-stream-mapmulti/